doc/MANUAL.md

```diff
@@ -259,6 +262,16 @@ A major benefit of using this NVD API is incremental updates which basically mea
 
 You may also choose to update the data using `json` option which uses the JSON feeds available on [this page](https://nvd.nist.gov/vuln/data-feeds). These per-year feeds are updated once per day.  This mode was the default for CVE Binary Tool prior to the 3.0 release.
 
+### --nvd-api-key NVD_API_KEY
+
+An NVD API key allows registered users to make a greater number of requests to the API.  At this time, the [NVD API documentation](https://nvd.nist.gov/developers)) says, "The public rate limit (without an API key) is 10 requests in a rolling 60 second window; the rate limit with an API key is 100 requests in a rolling 60 second window."
+
+CVE Binary tool by default queries the NVD database once per day and caches the results to help alleviate load on the NVD servers.  Users who update more regularly or who are running the tool in shared environments (such as cloud providers or GitHub Actions) may find themselves hitting the rate limits despite those precautions and should obtain and use an NVD API key with CVE Binary Tool.
+
+To get an API key, users should visit the [NVD API key request page](https://nvd.nist.gov/developers/request-an-api-key).
+
+Related: [NVD API Key Announcement](https://nvd.nist.gov/general/news/API-Key-Announcement)
```

Comment:
Just a nitpick. "Key" is in small letters in other instances except here. Maybe we should stick with either of one.

Reply:
Good nitpick.  I was going to claim these were page titles and should be capitalized, but as it turns out I didn't actually directly copy the titles so there's no need.  I'll make them both lowercase.